sources/packages/backend/workflow/test/2023-2024/parttime-assessment/awards-amounts/parttime-assessment-2023-2024-awards-amount-SBSD.e2e-spec.ts

```diff
@@ -0,0 +1,106 @@
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.programYearTotalSBSD).toBe(90);
+    expect(
+      calculatedAssessment.variables.dmnPartTimeAwardAllowableLimits
+        .limitAwardSBSD40AndUpCourseLoadAmount,
+    ).toBe(800);
+    expect(calculatedAssessment.variables.provincialAwardNetSBSDAmount).toBe(
+      710,
+    );
+    expect(
+      calculatedAssessment.variables.finalProvincialAwardNetSBSDAmount,
+    ).toBe(calculatedAssessment.variables.provincialAwardNetSBSDAmount);
+  });
+
+  it("Should determine provincialAwardSBSDAmount when awardEligibilitySBSD is true and offeringCourseLoad is < 40", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+    assessmentConsolidatedData.offeringCourseLoad = 39;
+
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.programYearTotalSBSD).toBe(90);
+    expect(
+      calculatedAssessment.variables.dmnPartTimeAwardAllowableLimits
+        .limitAwardSBSDUnder40CourseLoadAmount,
+    ).toBe(400);
+    expect(calculatedAssessment.variables.provincialAwardNetSBSDAmount).toBe(
+      310,
+    );
+    expect(
+      calculatedAssessment.variables.finalProvincialAwardNetSBSDAmount,
+    ).toBe(calculatedAssessment.variables.provincialAwardNetSBSDAmount);
+  });
+
+  it("Should determine provincialAwardNetSBSDAmount as zero when awardEligibilitySBSD is true, programYearTotalSBSD (programYearTotalFullTimeSBSD + programYearTotalPartTimeSBSD) is greater than the dmnPartTimeAwardAllowableLimits for SBSD ", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+    assessmentConsolidatedData.programYearTotalFullTimeSBSD = 500;
+    assessmentConsolidatedData.programYearTotalPartTimeSBSD = 600;
+
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.awardEligibilitySBSD).toBe(true);
+    expect(calculatedAssessment.variables.programYearTotalSBSD).toBe(1100);
+    expect(
+      calculatedAssessment.variables.dmnPartTimeAwardAllowableLimits
+        .limitAwardSBSD40AndUpCourseLoadAmount,
+    ).toBe(800);
+    expect(calculatedAssessment.variables.provincialAwardNetSBSDAmount).toBe(0);
+    expect(
+      calculatedAssessment.variables.finalProvincialAwardNetSBSDAmount,
+    ).toBe(calculatedAssessment.variables.provincialAwardNetSBSDAmount);
+  });
+
+  it("Should determine provincialAwardNetSBSDAmount as zero when awardEligibilitySBSD is false", async () => {
```

Comment:
This test is checking award eligibility which seems to be redundant with existing award eligibility tests.

Reply:
Added to cover additional tests related to false scenario awards calculation

Reply:
I would still recommend to add this test to award eligibility, but considering it as not a blocker.